terraform/variables.tf

```
@@ -30,7 +31,7 @@ variable "tags" {
   description = "Tags to apply to all AWS resources created"
 
   default = {
-    Team        = "NCATS OIS - Development"
-    Application = "ansible-role-nessus testing"
+    Team        = "VM Fusion - Development"
+    Application = "skeleton-ansible-role-with-test-user testing"
```

Looks like you accidentally overwrote this:
```suggestion
    Application = "ansible-role-nessus testing"
```